daml-assistant/daml-helper/src/DA/Daml/Helper/Ledger.hs

```diff
@@ -0,0 +1,51 @@
+-- Copyright (c) 2019 Digital Asset (Switzerland) GmbH and/or its affiliates. All rights reserved.
+-- SPDX-License-Identifier: Apache-2.0
+module DA.Daml.Helper.Ledger (
```

Comment:
I like the separate module :-) 

I'd also be happy if all of the related logic for running the commands was in this module as well (the `runDeploy`, `runAllocateParty`, `runListParty`, etc). But happy to leave this refactoring for another PR. DamlHelper sure needs it at this point.